docs/content/operations-as-service/sla.md

```diff
@@ -0,0 +1,120 @@
+* Control Plane — the cluster core — is considered running in the HA mode if it has at least three instances
+
+## Availability Types
+
+To accurately identify types of degradations and the degree of their impact on the performance of services, we define several types of unavailability for which it is possible to set the parameters under the SLA.
+
+### Synthetic Availability
+
+Synthetic availability is the availability of running applications in the cluster and cluster components that are not related to the availability of the cluster's Control Plane, such as:
+
+* Applications that have been started before in containers that are still running
+* Kubernetes services - the presence of corresponding `iptables` rules
+* DNS - the ability to resolve hosts
+
+The SLA for Synthetic Availability is valid only if these basic conditions are met:
+
+* Cluster nodes continue to be available
+* There is network connectivity between nodes
+* The application continues to operate correctly
+
+At the same time, applications keep on running that meet the following conditions:
+
+* The application itself caches DNS data
+* The application can operate with no scheduler or autoscaler running as well as with no ability to add/edit service endpoints
+* No `kube-apiserver` availability is required for the application to operate
+
+This availability type does not require enabling the HA mode for components.
+
+### Router Availability
+
+Router availability is the availability of ingress router and its components - incoming HTTP requests are delivered to end applications.
+
+The router availability SLA does not cover the network availability that depends on the infrastructure.
+
+This availability type requires enabling the HA mode for components.
+
+### Node Group Availability
+
+In managed OpenShift, nodes are grouped together based on a purpose and managed as an integrated entity. All nodes in the group have the same metadata defined in the parameters of a `NodeGroup` resource based on labels, annotations, or taint fields. Such a group of nodes is called the Node Group.
+
+Node Group Availability indicates that each Node Group has at least `N-1` correctly functioning nodes, where `N` is the minimum number of nodes in NodeGroup as defined in the configuration.
+
+The Node Group Availability SLA does not apply to cases when a node fails due to improper operation of an application installed by the customer.
+
+This availability type does not require enabling the HA mode for components.
+
+### Control Plane Availability
+
+Control plane availability indicates that the deployment process in the cluster is working, and self-healing processes are running. It formulates the availability level of the main control OpenShift components:
+
+* `etcd`
+* `kube-apiserver`
+* `kube-controller-manager`
+* `kube-scheduler`
+* `kube-dns`
```

Comment:
we should include the openshift-api-server and oauth server; plz check the names with genesis team